store/postgres/tests/relational.rs

```diff
@@ -546,18 +562,28 @@ fn delete() {
         insert_entity(&conn, &layout, "Scalar", two);
 
         // Delete where nothing is getting deleted
-        let mut key = EntityKey::data(
+        let key = EntityKey::data(
             THINGS_SUBGRAPH_ID.clone(),
             "Scalar".to_owned(),
             "no such entity".to_owned(),
         );
-        let count = layout.delete(&conn, &key, 1).expect("Failed to delete");
+        let entity_type = EntityType::from("Scalar");
+        let mut entity_keys = vec![key];
+        let count = layout
+            .delete(&conn, entity_type.clone(), &entity_keys, 1)
+            .expect("Failed to delete");
```

Comment:
We should also have a test for deletion of more than one entity - this will also help test that we update the entity count correctly in that case. Getting that wrong is very hard to detect, so we need to have good tests for that.